src/panels/config/automation/add-automation-element-dialog.ts

```diff
@@ -317,7 +321,7 @@ class DialogAddAutomationElement extends LitElement implements HassDialog {
         const icon = domainIconWithoutDefault(domain);
         result.push({
           group: true,
-          icon,
+          iconPath: icon,
```

Comment:
```suggestion
          icon: html`<ha-svg-icon .path=${item.iconPath} ></ha-svg-icon>`,
```
So we don't need the `iconPath` property.